bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBase.java

```diff
@@ -166,8 +166,14 @@ protected void sendResponseAndWait(int rc, Object response, OpStatsLogger statsL
 
     @Override
     public void run() {
-        requestProcessor.getRequestStats().getWriteThreadQueuedLatency()
-                .registerSuccessfulEvent(MathUtils.elapsedNanos(enqueueNanos), TimeUnit.NANOSECONDS);
+        if (this instanceof ReadEntryProcessor) {
```

Comment:
Use `request instanceof BookieProtocol.ReadRequest`?

Reply:
OK. It makes senses to me. This could uniform the code style. Updated @hangc0276